### DISCUSSION TITLE
Render Thousands Seperator during Input (similar to QDecimal)

### DISCUSSION THREAD

**Discussion by b0ot** [dm4t2/vue-currency-input#328] [Ideas]
QDecimal 
GitHub: https://github.com/TobyMosque/app-extension-qdecimal
Example: https://qdecimal.netlify.app/#/

Is able to render the final number as you type it into the box with the formatting. Currently, the Vue-Currency-Input removes rendering as you select it to a raw number. Once you deselect the input it adds the formatting. 

Example: vue-currency-input (typing in $1,000,000)
Select Input
-------------
1
10
100
1000
10000
1000000
--------------
Select off input
$1,000,000
============

Example: QDecimal (typing in $100,000) 
Select Input
-------------
$ 1
$ 10
$ 100
$ 1,000
$ 10,000
$ 100,000
$ 1,000,000
--------------
Select off input (already fully rendered)
$1,000,0000
============
![currency](https://user-images.githubusercontent.com/9140903/169645929-0ae94b1a-0374-4da1-9be4-ec3c6c8dd5b5.gif)

**Comment by dm4t2** [comment]
The desired input behavior is already covered by using `hideGroupingSeparatorOnFocus: false` and `autoDecimalDigits: true`, isn't it?

https://dm4t2.github.io/vue-currency-input/config.html#autodecimaldigits
https://dm4t2.github.io/vue-currency-input/config.html#hidegroupingseparatoronfocus

**Comment by b0ot** [comment]
I definitely missed that option. I figured it would be the default if it
was able to support it.
Thank you for linking exactly what I was looking for!
